src/containers/SubmitSteps/RecordingsSteps/Introduction/Introduction.tsx

```diff
@@ -159,11 +162,21 @@ const Introduction = ({
           <BulletIndicator>1</BulletIndicator>
         </WelcomeBullets>
         <BlackText>
-          <Trans i18nKey="recordingsIntroduction:recordCough.intro1">
-            Find a <strong>quiet environment</strong> at least
-            <strong>20 ft (6m)</strong> away from others and wear a cloth or surgical mask.
-            If you are feeling ill, please sit down.
-          </Trans>
+          {
+            isBreathLogic || isCoughLogic ? (
+              <Trans i18nKey="recordingsIntroduction:recordCough.intro1">
+                Find a <strong>quiet environment</strong> at least
+                <strong>20 ft (6m)</strong> away from others and wear a cloth or surgical mask.
+                If you are feeling ill, please sit down.
+              </Trans>
+            ) : (
+              <Trans i18nKey="recordingsIntroduction:recordBreath.intro1">
+                Find a <strong>quiet environment</strong> at least
+                <strong>20 ft (6m)</strong> away from others and wear a cloth or surgical mask.
+                If you are feeling ill, please sit down.
+              </Trans>
+            )
```

Comment:
What's the difference between two? 
Also, we are checking `isBreathLogic` for `then` and using `recordingsIntroduction:recordBreath.intro1` on the `else`